cernopendata/modules/fixtures/data/articles/about/md/cms.md

```diff
@@ -0,0 +1,75 @@
+
+# About CMS
```

Comment:
I don't like very much the structure

- `data/articles/about/md/cms.md`

when `md` is exposed in the path as a separate subdirectory... 

Better to have `/articles/<id>.json`, `/articles/<id>-body.md`, `/articles/<id>-fig1.png` perhaps?

Can muse IRL...

Reply:
My thinking was that after a while it would become a mess, since a lot more records are coming and if with every new one there are files attached, they wouldn't be that manageable.

Lets talk IRL